scylla-cql/src/frame/request/options.rs

```diff
@@ -11,3 +11,19 @@ impl SerializableRequest for Options {
         Ok(())
     }
 }
+
+/* Key names for options in SUPPORTED/STARTUP */
+pub const SCYLLA_SHARD_AWARE_PORT: &str = "SCYLLA_SHARD_AWARE_PORT";
+pub const SCYLLA_SHARD_AWARE_PORT_SSL: &str = "SCYLLA_SHARD_AWARE_PORT_SSL";
+
+pub const COMPRESSION: &str = "COMPRESSION";
+pub const CQL_VERSION: &str = "CQL_VERSION";
+pub const DRIVER_NAME: &str = "DRIVER_NAME";
+pub const DRIVER_VERSION: &str = "DRIVER_VERSION";
+
+/* Value names for options in SUPPORTED/STARTUP */
+pub const DEFAULT_CQL_PROTOCOL_VERSION: &str = "4.0.0";
+pub const DEFAULT_DRIVER_NAME: &str = "scylla-rust-driver";
+// TODO: consider using `env!`, which raises compile-time error if constant is not defined.
+// I think the constant should be always defined, no need to allow for it to be undefined.
```

Comment:
Docs agree with you: https://doc.rust-lang.org/cargo/reference/environment-variables.html#environment-variables-cargo-sets-for-crates
If the version is not in the manifest then the variable will be set to an empty string. I think you can use `env!`